lib/tasks/b2b.rake

```diff
@@ -0,0 +1,19 @@
+namespace :b2b do
+  desc 'Create corporate article type'
+  task create_type: :environment do
+    Rails.logger = ActiveRecord::Base.logger = Logger.new(STDOUT)
```

Comment:
Yeah, I always use this type of thing and always put some file to be write the log. Then I changed to STDOUT because was not so critical. I will use puts.